wallet/src/types.rs

```diff
@@ -198,6 +202,12 @@ pub struct ExtendedKeyedSignature {
 #[derive(Clone, Debug, Deserialize)]
 pub struct ChainEntry(pub u32, pub String);
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum BirthDate {
+    Current,
```

Comment:
You can use serde rename to have a more idiomatic API, if you want

```suggestion
    #[serde(rename = "current")]
    Current,
```